html-forms/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <div>
+      Email:  <br />
+      <label for="email"><input type="text" id="email"></label>
```

Comment:
You would want the text that is labelling the inputs to be inside of the `label`, rather than above or outside of the element.